ticker/src/main/java/com/robinhood/ticker/TickerColumn.java

```diff
@@ -209,6 +212,12 @@ private boolean drawText(Canvas canvas, Paint textPaint, char[] characterList, i
         if (index >= 0 && index < characterList.length) {
             canvas.drawText(characterList, index, 1, 0f, verticalOffset, textPaint);
             return true;
+        } else if (startIndex == UNKNOWN_START_INDEX && index == UNKNOWN_START_INDEX) {
```

Comment:
Yep handles it fine, the two unknowns will just be next to each other (this is why I used two constants to define unknowns for start and end instead of sharing one constant for both) and animate accordingly.